config/jenkins-jobs_infra.ci.jenkins.io.yaml

```diff
@@ -523,3 +523,8 @@ jobsDefinition:
             description: "stats.jenkins.io File Share Service Principal Writer"
             subscriptionId: "${JENKINSINFRA_AZURE_PRIMARY_SUBSCRIPTION_ID}"
             tenant: "${JENKINSINFRA_AZURE_TENANT_ID}"
+      stats.jenkins.io_previews:
```

Comment:
- The `website-jobs` directory has a scoped credential for the Netlify preview publication which is applied all of its children
- The current job `stats.jenkins.io` has a specific credential for production

=> as such, it would be safer (due to credential scoping) and more logical to move the "production" deployment to another folder than `website-jobs`